profiles/schema.py

```diff
@@ -50,10 +56,60 @@ def resolve_type(self, info, **kwargs):
     )
 
 
+class ProfilesConnection(graphene.Connection):
+    class Meta:
+        abstract = True
+
+    count = graphene.Int()
+    total_count = graphene.Int()
+
+    def resolve_count(self, info):
+        return self.length
+
+    def resolve_total_count(self, info):
+        return self.iterable.model.objects.filter(
+            service__service_type=SERVICE_TYPES[1][0]
+        ).count()
+
+
+class ProfileFilter(FilterSet):
+    class Meta:
+        model = Profile
+        fields = ("first_name", "last_name", "nickname", "email", "phone", "language")
+
+    first_name = CharFilter(lookup_expr="icontains")
+    last_name = CharFilter(lookup_expr="icontains")
+    nickname = CharFilter(lookup_expr="icontains")
+    email = CharFilter(lookup_expr="icontains")
+    phone = CharFilter(lookup_expr="icontains")
+    language = CharFilter(lookup_expr="icontains")
```

Comment:
maybe it should be `exact`? I am not sure, but maybe. 🤔 it is default, so if you agree, you can just do `CharFilter()` here on this line.